Test failing with PYPI package too:
>    self.assertTrue(cleared)
> AssertionError: False is not true


diff --git a/test/dynamo/test_misc.py b/test/dynamo/test_misc.py
index 7541bd3b9d8..d0cb310bec6 100644
--- a/test/dynamo/test_misc.py
+++ b/test/dynamo/test_misc.py
@@ -10992,6 +10992,7 @@ fn
             lambda mod: mod,
         )
 
+    @unittest.skip("Unreliable")
     def test_outside_linear_module_free(self):
         # Compared to test_linear_module_free, the linear
         # layer is not the code object that is directly compiled.
@@ -11026,6 +11026,7 @@ fn
         gc.collect()
         self.assertTrue(cleared)
 
+    @unittest.skip("Unreliable")
     def test_parameter_free(self):
         def model_inp_ctr():
             param = torch.nn.Parameter(torch.randn(100, 100))
